dashboard/test/models/script_test.rb

```diff
@@ -1131,6 +1131,118 @@ def populate_cache_and_disconnect_db
+
+      @unit_teacher_to_students = create(:script, instructor_audience: SharedCourseConstants::INSTRUCTOR_AUDIENCE.teacher, participant_audience: SharedCourseConstants::PARTICIPANT_AUDIENCE.student)
+      @unit_facilitator_to_teacher = create(:script, instructor_audience: SharedCourseConstants::INSTRUCTOR_AUDIENCE.facilitator, participant_audience: SharedCourseConstants::PARTICIPANT_AUDIENCE.teacher)
+      @unit_code_instructor_to_teacher = create(:script, instructor_audience: SharedCourseConstants::INSTRUCTOR_AUDIENCE.code_instructor, participant_audience: SharedCourseConstants::PARTICIPANT_AUDIENCE.teacher)
+      @unit_plc_reviewer_to_facilitator = create(:script, instructor_audience: SharedCourseConstants::INSTRUCTOR_AUDIENCE.plc_reviewer, participant_audience: SharedCourseConstants::PARTICIPANT_AUDIENCE.facilitator)
+      @unit_code_instructor_to_teacher = create(:script, instructor_audience: SharedCourseConstants::INSTRUCTOR_AUDIENCE.code_instructor, participant_audience: SharedCourseConstants::PARTICIPANT_AUDIENCE.teacher)
+    end
+
+    test 'unit in course should check course for participant and instructor audience' do
+      assert_equal @unit_group.can_be_instructor?(@teacher), @unit_in_course.can_be_instructor?(@teacher)
+      assert_equal true, @unit_in_course.can_be_instructor?(@teacher)
+
+      assert_equal @unit_group.can_be_participant?(@teacher), @unit_in_course.can_be_participant?(@teacher)
+      assert_equal true, @unit_in_course.can_be_participant?(@student)
+    end
+
+    test 'code instructor should be able to instruct any unit' do
+      assert_equal true, @unit_teacher_to_students.can_be_instructor?(@code_instructor)
+      assert_equal true, @unit_facilitator_to_teacher.can_be_instructor?(@code_instructor)
+      assert_equal true, @unit_code_instructor_to_teacher.can_be_instructor?(@code_instructor)
+      assert_equal true, @unit_plc_reviewer_to_facilitator.can_be_instructor?(@code_instructor)
+      assert_equal true, @unit_code_instructor_to_teacher.can_be_instructor?(@code_instructor)
+    end
+
+    test 'levelbuilder should be able to see instructor view for any unit' do
+      assert_equal true, @unit_teacher_to_students.can_be_instructor?(@levelbuilder)
+      assert_equal true, @unit_facilitator_to_teacher.can_be_instructor?(@levelbuilder)
+      assert_equal true, @unit_code_instructor_to_teacher.can_be_instructor?(@levelbuilder)
+      assert_equal true, @unit_plc_reviewer_to_facilitator.can_be_instructor?(@levelbuilder)
+      assert_equal true, @unit_code_instructor_to_teacher.can_be_instructor?(@levelbuilder)
+    end
+
+    test 'plc reviewer should be able to instruct units with plc_reviewer as instructor audience ' do
+      # Since the plc reviewer is a teacher account it will also be able to teach any teacher course
+      assert_equal true, @unit_teacher_to_students.can_be_instructor?(@plc_reviewer)
+      assert_equal false, @unit_facilitator_to_teacher.can_be_instructor?(@plc_reviewer)
```

Comment:
You could just do 
`assert @unit_teacher_to_students.can_be_instructor?(@plc_reviewer)` and 
`refute  @unit_facilitator_to_teacher.can_be_instructor?(@plc_reviewer)` 
here and elsewhere rather than `assert_equal true`... But I think it's a stylistic preference.

Reply:
Lol yes I'm not sure why this is where my brain went. I will update